xenmgr/Vm/Config.hs

```diff
@@ -852,6 +854,8 @@ miscSpecs cfg = do
                                 _       -> case name of
                                              "viridian" -> name ++ "=" ++ (wrapBrackets $ wrapQuotes v)
                                              "serial"   -> name ++ "=" ++ (wrapBrackets $ wrapQuotes v)
+                                             "iomem"   -> name ++ "=" ++ (wrapBrackets $ wrapQuotes v)
```

Comment:
This naively only supports a single value when iomem can handle multiple ranges:
iomem=[ "IOMEM_START,NUM_PAGES[@GFN]", "IOMEM_START,NUM_PAGES[@GFN]", ... ]

You can jam multiple values in by writing something like "0xfed40,4','0xfff60,1" so the inner single quotes are passed through and interpreted by xl.